Packs/Workday/Integrations/WorkdayIAM/WorkdayIAM.py

```diff
@@ -0,0 +1,294 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+''' IMPORTS '''
+import traceback
+import requests
+from datetime import datetime, timedelta
+
+LAST_DAY_OF_WORK_EVENT_FIELD = 'lastdayofwork'
+TERMINATION_DATE_EVENT_FIELD = 'terminationdate'
+EMPLOYMENT_STATUS_EVENT_FIELD = 'employmentstatus'
+HIRE_DATE_EVENT_FIELD = 'hiredate'
+EMPLOYEE_ACTIVE_STATUS = 'active'
+WORKDAY_DATE_TIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+WORKDAY_DATE_FORMAT = "%m/%d/%Y"
+READ_TIME_OUT_IN_SECONDS = 300
+INCIDENT_TYPE = 'IAM-Sync-User'
+DEFAULT_MAPPER_IN = 'IAM Sync User - Workday'
+
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, should not contain Cortex XSOAR logic.
+    Should do requests and return data
+    """
+    def __init__(self, base_url: Any, verify: bool, headers: dict, proxy: bool, ok_codes: tuple,
+                 auth: Any, report_url: str):
+        super().__init__(base_url=base_url, verify=verify, headers=headers, proxy=proxy, ok_codes=ok_codes, auth=auth)
+        self._report_url = report_url
```

Comment:
no need to have the url as part of the client. Pass it to fetch_incidents and other commands as argument